react/utils/showAlertOptions.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line no-shadow
```

Comment:
Is this needed? I tried to search about the error but it didnt make much sense to me. So i'm not sure about it.

Reply:
demorou um pouco, mas descobri o porque dessa linha
antes era porque eu levava um erro de lint
mas eu atualizei ele e agora não é mais necessário :)